server/polar/subscription/endpoints.py

```diff
@@ -129,3 +138,73 @@ async def create_csv() -> AsyncGenerator[str, None]:
         media_type="text/csv",
         headers={"Content-Disposition": f"attachment; filename={filename}"},
     )
+
+
+@router.patch(
+    "/{id}",
+    summary="Update Subscription",
+    response_model=SubscriptionSchema,
+    responses={
+        200: {"description": "Subscription updated."},
+        403: {
+            "description": (
+                "Subscription is already canceled "
+                "or will be at the end of the period."
+            ),
+            "model": AlreadyCanceledSubscription.schema(),
+        },
+        404: SubscriptionNotFound,
+    },
+)
+async def update(
+    id: SubscriptionID,
+    subscription_update: SubscriptionUpdate,
+    auth_subject: auth.SubscriptionsWrite,
+    session: AsyncSession = Depends(get_db_session),
+) -> Subscription:
+    """Update a subscription of the authenticated customer or user."""
```

Comment:
Smells like a copy/paste 😊 Just `Update a subscription.` in this context.

Reply:
Haha guilty 😅 Nice catch